src/timeMachine/components/view_options/BandOptions.tsx

```diff
@@ -291,17 +347,37 @@ class BandOptions extends PureComponent<Props> {
 
   private onChangeLowerColumn = selectedLowerColumnName =>
     this.onChangeColumn(this.props.onSetLowerColumn, selectedLowerColumnName)
+
+  private onTextChangeUpperColumn = (e: ChangeEvent<HTMLInputElement>) => {
```

Comment:
there's very little cost to adding `vent` to `e` (to make it `event`). `e` is really ambiguous in javascript - it often represents `event` or `error`. being able to search for instances of event handlers is much easier when they're named `event` or `evt`. try doing a global search for `e` or `e:`.

Reply:
Originally, i wanted event, but for some reason my VS Code put a strikethrough over `event`, which puzzled me. That's when I changed it to `e`. Let me see if changing it back now has the same issue.

Reply:
okay it worked this time, no complaints from VS Code.